model/metrics/v8js-metrics.yaml

```
@@ -0,0 +1,96 @@
+groups:
+  - id: metric.veightjs.gc.duration
+    type: metric
+    metric_name: v8js.gc.duration
+    brief: "Garbage collection duration."
+    instrument: histogram
+    unit: "s"
+    stability: experimental
+    prefix: v8js.gc
+    attributes:
+      - ref: v8js.gc.type
+        requirement_level: recommended
+
+  - id: metric.veightjs.memory.total
+    type: metric
+    metric_name: v8js.memory.total
```

```suggestion
    metric_name: v8js.memory.limit
```

similar to [jvm.memory.limit](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/runtime/jvm-metrics.md#metric-jvmmemorylimit) and also 

> limit - an instrument that measures the constant, known total amount of something should be called entity.limit. For example, system.memory.limit for the total amount of memory on a system.

https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#instrument-naming